govc/host/esxcli/command.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/govmomi/internal"
```

Comment:
@dougm if I change it to `github.com/vmware/govmomi/vim25/internal` then it's not importable from here in `github.com/vmware/govmomi/govc`

internal package spec described here: https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit

Reply:
we can do `internal/vim25` though. it's not conflicting with methods/types, etc.

Reply:
ok thanks @anfernee , let's stick with the top-level internal for now.